data/json/items/comestibles.json

```diff
@@ -12210,6 +12210,36 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "frenchtoast",
+        "name" : "French toast",
+        "name_plural" : "French toasts",
+        "weight" : 246,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 240,
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 0,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 45,
+        "description" : "Slices of bread dipped in a milk and egg mixture then fried.",
+        "price" : 600,
+        "material" : "wheat",
```

Comment:
Should have egg and milk in the materials too, otherwise it won't be allergic to herbivores and lactose intolerant.